tools/serverpod_cli/lib/src/generator/dart/library_generators/class_generators/repository_classes.dart

```diff
@@ -210,10 +210,19 @@ class BuildRepositoryClass {
       Iterable<SerializableModelFieldDefinition> objectRelationFields) {
     return Method((m) => m
       ..docs.add(
-          '/// Find a list of [$className]s from a table, using the provided [where]\n'
-          '/// expression, optionally using [limit], [offset], and [orderBy]. To order by\n'
-          '/// multiple columns, use [orderByList]. If [where] is omitted, all rows in\n'
-          '/// the table will be returned.')
+          '/// Returns a list of [$className]s matching the given query parameters.\n'
+          '/// \n'
+          '/// Use [where] to specify which items to include in the return value.\n'
+          '/// If none is specified, all items will be returned.\n'
+          '///\n'
+          '/// To specify the order of the items use [orderBy] or [orderByList]\n'
+          '/// when sorting by multiple columns.\n'
+          '///\n'
+          '/// The maximum number of items can be set by [limit]. If no limit is set,\n'
+          '/// all items matching the query will be returned.\n'
+          '///\n'
+          '/// [offset] defines how many items to skip, after which [limit] (or all)\n'
+          '/// items are read from the database.')
```

Comment:
Suggestion: You could also do this as a multiline string and you can remove the \n e.g: 

```dart
'''/// Returns a list of [$className]s matching the given query parameters.
///
/// Use [where] to specify which items to include in the return value.
/// If none is specified, all items will be returned.
'''
```

Reply:
Right, in the end this will probably read better even if the first line will be off (let's see, maybe even having an empty-line start would work fine here, since the code should get formatted in the end and thus there would be no double blank line).

Will try this out.

Reply:
Switched to multiline. Looks much better indeed.